packages/flutter_test/lib/src/accessibility.dart

```diff
@@ -283,29 +296,36 @@ class MinimumTextContrastGuideline extends AccessibilityGuideline {
 
   @override
   Future<Evaluation> evaluate(WidgetTester tester) async {
-    final SemanticsNode root = tester.binding.pipelineOwner.semanticsOwner!.rootSemanticsNode!;
-    final RenderView renderView = tester.binding.renderView;
-    final OffsetLayer layer = renderView.debugLayer! as OffsetLayer;
-
-    late ui.Image image;
-    final ByteData? byteData = await tester.binding.runAsync<ByteData?>(
-      () async {
-        // Needs to be the same pixel ratio otherwise our dimensions won't match
-        // the last transform layer.
-        final double ratio = 1 / tester.binding.window.devicePixelRatio;
-        image = await layer.toImage(renderView.paintBounds, pixelRatio: ratio);
-        return image.toByteData();
-      },
-    );
+    Evaluation result = const Evaluation.pass();
+    for (final FlutterView view in tester.platformDispatcher.views) {
+      // TODO(pdblasi): This renderView will need to be retrieved from view when available.
+      final RenderView renderView = tester.binding.renderView;
+      final OffsetLayer layer = renderView.debugLayer! as OffsetLayer;
+      final SemanticsNode root = tester.binding.pipelineOwner.semanticsOwner!.rootSemanticsNode!;
```

Comment:
nit: you can get the root node from the renderView above via renderView.owner.semanticsOwner.rootSemanticsNode. If we do that, the only TODO left here is to get the right renderView for the current view. Otherwise, this line probably also needs to get a TODO to get the right root node.

Reply:
Done.